manage/manage.js

```diff
@@ -100,7 +101,20 @@ newUI.renderClass();
 
 msg.onExtension(onRuntimeMessage);
 window.on('closeOptions', () => router.updateHash(''));
-router.watch({hash: '#stylus-options'}, toggleEmbeddedOptions);
+router.watch(
```

Comment:
This is bugged:
1. open manager
2. click Options
3. click `x` to close Options

The URL will change back, but the options UI won't disappear.